models/safe/ethereum/safe_ethereum_sources.yml

```diff
@@ -169,6 +169,21 @@ sources:
           - &singleton
             name: singleton
             description: "Singleton"
+      - name: SafeProxyFactory_v1_4_1_evt_ProxyCreation
+        loaded_at_field: evt_block_time
+        description: "Safe v1.4.1 factory ProxyCreation events"
+        columns:
+          - *contract_address
+          - *evt_block_number
+          - *evt_block_time
+          - *evt_index
+          - *evt_tx_hash
+          - &proxy
```

Comment:
this `&` character, i.e. anchor in the error message, is already used on this field above in the schema file. you can't use it twice on the same field. instead, you'd use `*` to refer to above reference of the same column name, or just remove usage of either character and add column with just name/description if the properties on the column are different for whatever reason